linpack/linpack_extra/run_linpack.sh

```diff
@@ -171,23 +171,25 @@ process_summary()
 	# We will over write this
 	#
 	echo Failed > test_results_report
+#	ls | rev | cut -d'_' -f1-14 | rev > /tmp/linpack_temp
 	ls | cut -d'.' -f1-5 | sort -u > /tmp/linpack_temp
 	iters=0
 	input="/tmp/linpack_temp"
 
-	$TOOLS_BIN/test_header_info --front_matter --results_file results_${test_name}.csv  $to_configuration --sys_type $to_sys_type --tuned $to_tuned_setting --results_version $test_version --test_name $test_name
-	echo hyper_config:sockets:threads:unit:"MB/sec:cpu_affin" >> results_${test_name}.csv
+	$TOOLS_BIN/test_header_info --front_matter --results_file results.csv  $to_configuration --sys_type $to_sys_type --tuned $to_tuned_setting --results_version $test_version --test_name $test_name
+	echo hyper_config:sockets:threads:unit:"MB/sec:cpu_affin" >> results.csv
 
 	while IFS= read -r lin_file
 	do
 		ls  ${lin_file}* > /tmp/linpack_files_iter
 		input1="/tmp/linpack_files_iter"
+		values=`ls | rev | cut -d'_' -f1-14 | rev`
 		iters=0
 		sum=0
-#linpack.20k.out.test_m5.xlarge_threads_1_sockets_1_hyper_no.iter_1
-		threads=`echo $lin_file | cut -d'_' -f 4`
-		sockets=`echo $lin_file | cut -d'_' -f 6`
-		hyper_setting=`echo $lin_file | cut -d'_' -f7-10 | cut -d'.' -f 1`
+#threads_128_sockets_2_hyper_yes_2_socket_hyp_throughput-performance_numa_interleave_all.iter_1
```

Comment:
We really should have called this "ht" instead of "hyper", but if that breaks other tools then let's leave it alone.  Also, if this is supposed to be an example of what it'll look like it'd be better if we expand it to actually make sense to the neophyte.  <seagull mode="engaged">

Reply:
Replace hyper with ht through out code.  Also removed unused variables.